content/components/underline-nav.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Underline nav
+description: The UnderlineNav is used to display primary navigation.
+componentId: underline_nav
+reactId: underline_nav
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=18843-67449&t=XqqSolC6AzRO9nae-11
+rails: https://primer.style/view-components/components/alpha/underlinenav/
+---
+
+# Underline nav
+
+The underline nav is used to provide navigation links to related views for the user's current context. It is typically used as primary navigation at the top of the page or as secondary navigation when nested in views with a [nav list](/components/nav-list).
```

Comment:
Non-blocking nitpick: "at the top of the page" could be misleading. In dotcom, the top of the page has the global GitHub header bar and navigation.